Prox/Prox/PlaceDetails/PlaceDetailsCardViewController.swift

```diff
@@ -122,6 +119,47 @@ class PlaceDetailsCardViewController: UIViewController {
         }
     }
 
+    private func setupCardInteractions() {
+        cardView.urlLabel.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(openPlaceURL(gestureRecgonizer:))))
+        cardView.yelpReviewView.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(openYelpReview(gestureRecgonizer:))))
+        cardView.tripAdvisorReviewView.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(openTripAdvisorReview(gestureRecgonizer:))))
+        cardView.travelTimeView.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(openDirections(gestureRecgonizer:))))
+    }
+
+    @objc private func openPlaceURL(gestureRecgonizer: UITapGestureRecognizer) {
```

Comment:
What broke was me removing the `isUserinteractionEnabled = true` flag from the URL label which prevented the `UITapGestureRecognizer` from being fired. This code works just fine.